Docs: Angular integration update. (#8944)
ckeditor/ckeditor5: docs/builds/guides/frameworks/angular.md

Because of the breaking changes in the Angular library output format, the `ckeditor5-angular` package is released in the following versions to support various Angular ecosystems:

> * Versions `1.x.x` &ndash; For **Angular 5-8** applications. We no longer provide support for these, as this version of Angular is not supported anymore.

Reviewer: ```suggestion
* Versions `1.x.x` &ndash; For **Angular 5-8** applications. We no longer provide support for these as the official support for these Angular versions was dropped.
```
